cachi2/core/models/property_semantics.py

```diff
@@ -78,4 +91,5 @@ def merge(self, other: "Self") -> "Self":
             missing_hash_in_file=self.missing_hash_in_file | other.missing_hash_in_file,
             npm_bundled=self.npm_bundled and other.npm_bundled,
             npm_development=self.npm_development and other.npm_development,
+            pip_package_binary=self.pip_package_binary or other.pip_package_binary,
```

Comment:
Should we enhance the test_property_semantics.py::test_merge unit test with this new case?

Reply:
Probably.

Reply:
...as well as `test_merge_component_properties`

Reply:
Fixed